templates/user/settings/profile.html.twig

```diff
@@ -43,6 +43,41 @@
             }}) }}
             {{ form_row(form.avatar, {label: 'avatar'}) }}
             {{ form_row(form.cover, {label: 'cover'}) }}
+
```

Comment:
Keep using `form_row()` if you want to add additional fields/rows to the form, just like above. So it stays consistent.